xml/security-horizon_ssl_auth.xml

```diff
@@ -212,7 +212,7 @@
      The following is an example of a properly formatted DN for a certificate
      issuer named "intermedia".
     </para>
-<screen>/C=US/ST=California/O=HPE/OU=Engineering/CN=intermediateca/emailAddress=intermediateca@hpe.com</screen>
+<screen>/C=US/ST=California/O=SUSE/OU=Engineering/CN=intermediateca/emailAddress=intermediateca@hpe.com</screen>
```

Comment:
Don't know about this one -- we usually use example companies instead of "SUSE". Also there is the discrepancy between "intermedia"/"intermediateca" (Spanish??) and the mail address is still \@hpe.com.
Ordinarily, at least for the mail we'd use something like maybe "certificates\@example.org" and the company name could be "Example, LLC" perhaps. (Please adapt my suggestions where they don't make sense.)

If you really want to use a real organization, you could also use `&vendor;` which will switch between SUSE and HPE depending on document variant.